hw/ip/aes/pre_sca/alma/cpp/testbench.h

```diff
@@ -0,0 +1,74 @@
+// Copyright lowRISC contributors.
+// Copyright IAIK.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_HW_IP_AES_PRE_SCA_ALMA_CPP_TESTBENCH_H_
+#define OPENTITAN_HW_IP_AES_PRE_SCA_ALMA_CPP_TESTBENCH_H_
+
+#include "verilated.h"
+#include "verilated_vcd_c.h"
+
+template <class Module>
+struct Testbench {
+  unsigned long m_tickcount;
+  Module m_core;
+  VerilatedVcdC *m_trace;
+
+  Testbench() {
+    Verilated::traceEverOn(true);
+    Module m_core;
```

Comment:
Sorry, I wasn't clear in my previous comment. You can delete this line entirely (the `m_core` field gets constructed automatically).

This code actually says to create another instance of type `Module` as a local variable.

Reply:
Sorry, I probably wasn't paying close attention. I've removed it now.